go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/network/armnetwork v1.1.0
 	github.com/IBM-Cloud/bluemix-go v0.0.0-20230120122421-afb48116b8f1
 	github.com/IBM/go-sdk-core/v5 v5.10.2
-	github.com/IBM/vpc-go-sdk v0.32.0
+	github.com/IBM/vpc-go-sdk v1.0.1
```

Comment:
I think QiFeng purposely downgraded the vpc-go-sdk to the latest maintained one: https://github.com/confidential-containers/cloud-api-adaptor/pull/701#issuecomment-1463371224